src/components/organisms/OrchestrationTable/index.tsx

```diff
@@ -0,0 +1,151 @@
+                        </Typography>
+                        <IconButton>
+                          <KeyboardArrowDown />
+                        </IconButton>
+                      </Box>
+                    </TableCell>
+                  )
+                } else {
+                  return (
+                    <TableCell align="center" className="cell">
+                      <Typography sx={{ fontWeight: 'bold', fontSize: '14px' }}>
+                        {data}
+                      </Typography>
+                    </TableCell>
+                  )
+                }
+              })}
+          </TableRow>
+        </TableHead>
+        <TableBody>
+          {tableData &&
+            tableData?.data.map((data: any) => {
+              return (
+                <TableRow
+                  //   onClick={(e) => handleCheckBox(data.id)}
+                  key={data.id}
+                  className="row"
+                  sx={{
+                    backgroundColor: data.isChecked && '#d2d2d2',
+                    '&:hover': {
+                      backgroundColor: data.isChecked ? '#d2d2d2' : ' #ebebeb;',
+                    },
+                  }}
+                >
+                  <TableCell className="cell" align="center">
+                    <CheckBox
+                      sx={{ transform: 'scale(0.8)', color: '#c1c1c1' }}
+                      key={data.id}
+                      onChange={(e) => handleCheckBox(data.id)}
+                      checked={data.isChecked}
+                    />
+                  </TableCell>
+                  <TableCell className="cell" align="left">
+                    <Typography
+                      sx={{
+                        fontSize: '12px',
+                      }}
+                    >
+                      {data.journeyName}
+                    </Typography>
+                  </TableCell>
+                  <TableCell className="cell" align="left">
+                    <Typography
+                      sx={{
+                        fontSize: '12px',
+                      }}
+                    >
+                      {data.description}
+                    </Typography>
+                  </TableCell>
+                  <TableCell className="cell" align="center">
+                    <Typography
+                      sx={{
+                        fontSize: '12px',
+                      }}
+                    >
+                      {data.group}
+                    </Typography>
+                  </TableCell>
+                  <TableCell className="cell" align="center">
+                    <Typography
+                      sx={{
+                        fontSize: '12px',
+                      }}
+                    >
+                      {data.status}
+                    </Typography>
+                  </TableCell>
+                </TableRow>
+              )
+            })}
+        </TableBody>
+      </Table>
```

Comment:
Table should be a molecule, where it receives the props like rows, columns, etc., so that it will become reusable